app/src/main/java/com/animal_villa/GameFramework.kt

```diff
@@ -6,12 +6,13 @@ import androidx.activity.ComponentActivity
 import com.example.main_animal_villa.R
 //import java.lang.reflect.Modifier
 
-class GameFramework : ComponentActivity() {
+public class GameFramework : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
     }
 }
+// Changes made to indentation
```

Comment:
Made sure indentation follows kotlin coding conventions https://kotlinlang.org/docs/coding-conventions.html#indentation

Reply:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.